src/rendering/renderable.cpp

```diff
@@ -112,6 +113,22 @@ Renderable::Renderable(const ghoul::Dictionary& dictionary)
     dictionary.getValue(keyStart, _startTime);
     dictionary.getValue(keyEnd, _endTime);
 
+    std::string tagName;
+    ghoul::Dictionary tagNames;
+    if (dictionary.hasKeyAndValue<std::string>(KeyTag)) {
+        dictionary.getValue(KeyTag, tagName);
```

Comment:
Better (since you already checked the existence and type:
`std::string tagName = dictionary.value<std::string.(KeyTag);`